### PR TITLE
Doc tweaks

### PR DESCRIPTION
**Pull request by SrinivasanTarget**
## Proposed changes

Updated chromedriver path
## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._
- [x] I have read the [CONTRIBUTING](/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://docs.google.com/forms/d/1lOfXRw_0VCk7gYzjj4WLetGu7yelDVo5LWh0z7pGftE/viewform)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by moizjv** [comment]
Can one of the admins verify this patch?

**Comment by imurchie** [comment]
This needs a little more clarification. The path you give is only valid for npm 3+, which has a flat directory structure. In npm 2 it will be in `node_modules/appium-android-driver/node_modules/appium-chromedriver/chromedriver<os-name>`.

**Comment by SrinivasanTarget** [comment]
Updated as per comments now

File: docs/en/writing-running-appium/mobile-web.md [hunk 1]
```diff
@@ -176,7 +176,7 @@ class ContextTests extends PHPUnit_Extensions_AppiumTestCase
...
-*  If downloaded from [NPM](https://www.npmjs.org/package/appium), or running from the [.app](https://github.com/appium/appium-dot-app), nothing needs to be done. If running from source, `npm install` will download ChromeDriver and put it in `node_modules/appium-android-driver/node_modules/appium-chromedriver/chromedriver`. A particular version can be specified by passing the `--chromedriver_version` config property (e.g., `npm install appium --chromedriver_version="2.16"`), otherwise the most recent one will be retrieved.
+*  If downloaded from [NPM](https://www.npmjs.org/package/appium), or running from the [.app](https://github.com/appium/appium-dot-app), nothing needs to be done. If running from source, `npm install` will download ChromeDriver and put it in `node_modules/appium-chromedriver/chromedriver/<OS name>/` for users having npm v3+ and for npm v2 it will be in `node_modules/appium-android-driver/node_modules/appium-chromedriver/chromedriver<os-name>`. A particular version can be specified by passing the `--chromedriver_version` config property (e.g., `npm install appium --chromedriver_version="2.16"`), otherwise the most recent one will be retrieved.
```
Comment (imurchie) [inline docs/en/writing-running-appium/mobile-web.md]: Can you update so "OS name" is the same in both paths? I don't care which one (`OS name` or `os-name`). The npm 2 version needs a `/` before it, too. Sorry I mis-wrote before.

**Comment by SrinivasanTarget** [comment]
@imurchie Updated the same names for both npm versions

**Comment by imurchie** [comment]
Looks good. Thanks!

**Pull request merged**